scalableminds/webknossos
# Tracing statistics

As discussed, although this is put on hold for now, we'll merge this working script. We can always improve on it later.

### Steps to test:
- Start a server locally and run the script. If there are no results, edit the tracingTime of an annotation with a task in the db to a much higher value. Try again, and there should be a result.

### Issues:
- #2608

------
